docs/application/native/guides/text-input/autofill.md

```diff
@@ -0,0 +1,365 @@
+        char *view_id = NULL;
+
+        autofill_auth_info_get_view_id(auth_info_h, &view_id);
+        autofill_auth_info_get_autofill_data_present(auth_info_h, &autofill_data_present);
+        autofill_auth_info_get_authentication_needed(auth_info_h, &authentication_needed);
+
+        dlog_print(DLOG_INFO, LOG_TAG, "view id : %s, exist autofill data : %d, need authentication : %d", view_id, autofill_data_present, authentication_needed);
+
+        if (view_id)
+            free(view_id);
+
+        if (!autofill_data_present)
+            return;
+
+        if (authentication_needed)
+        {
+            autofill_auth_info_get_service_name(auth_info_h, &service_name);
+            autofill_auth_info_get_service_message(auth_info_h, &service_message);
+            autofill_auth_info_get_service_logo_image_path(auth_info_h, &service_logo_image_path);
+
+            dlog_print(DLOG_INFO, LOG_TAG, "service name : %s, logo path : %s, message : '%s'", service_name, service_logo_image_path, service_message);
+
+            /* create authentication popup */
+            ...
+
+            if (service_message)
+                free(service_message);
+
+            if (service_name)
+                free(service_name);
+
+            if (service_logo_image_path)
+                free(service_logo_image_path);
+        }
+    }
+
+    /* Set callback function for receiving authentication information */
+    autofill_auth_info_set_received_cb(ah, auth_info_received_cb, NULL);
+    ```
+
+2. Create autofill data for each input field and view information to include several autofill data:
+
+    ```c
+    int ret;
+    autofill_view_info_h vi_h;
+    autofill_item_h ai_h;
+    char *app_id = NULL;
+
+    app_get_id(&app_id);
+
+    /* create autofill item */
+    autofill_item_create(&ai_h);
+    autofill_item_set_autofill_hint(ai_h, AUTOFILL_HINT_ID);
+    autofill_item_set_id(ai_h, "id");
+    autofill_item_set_label(ai_h, "label");
+    autofill_item_set_sensitive_data(ai_h, false);
+
+    /* create autofill view info */
+    autofill_view_info_create(&vi_h);
+    autofill_view_info_set_app_id(vi_h, app_id);
+    autofill_view_info_set_view_id(vi_h, "login");
+
+    /* append autofill item in autofill view */
+    autofill_view_info_add_item(vi_h, ai_h);
+
+    if (app_id)
+        free(app_id);
+    ```
+
+3. Request to get the authentication information:
+
+    ```c
+    /* Send request to get authentication information */
+    ret = autofill_auth_info_request(ah, vi_h);
+    if (ret != AUTOFILL_ERROR_NONE)
+        /* Error handling */
+    ```
+
+4. When you no longer need the autofill view handle, call `autofill_view_info_destroy()` to release autofill view handle:
+
+    ```c
+    autofill_view_info_destroy(vi_h);
+    ```
+
+## Retrieving the data previously entered by the user and stored in an autofill service
+
+To Retrieve the data previously entered by the user and stored in an autofill service:
```

Comment:
To retrieve the data previously entered by the user and stored in an Autofill service: